submissions/jeffersonMormah/database/query.sql

```diff
@@ -0,0 +1,25 @@
+-- total number of survived passengers
+SELECT COUNT(*) FROM accidents WHERE survived = 1;
```

Comment:
kindly place the result of the count you got in a comment as stated in your task